tasks/settings.yml

```diff
@@ -8,6 +8,15 @@
     state: present
   register: jenkins_init_prefix
 
+- name: Add Java options to Jenkins launch command
+  lineinfile:
+    dest: "{{ jenkins_init_file }}"
+    insertafter: '^JENKINS_JAVA_OPTIONS='
+    regexp: '^JENKINS_JAVA_OPTIONS=\"\$JENKINS_JAVA_OPTIONS '
+    line: 'JENKINS_JAVA_OPTIONS="$JENKINS_JAVA_OPTIONS {{ jenkins_java_options }}"'
+    state: present
+  register: jenkins_init_prefix
```

Comment:
Since this overwrites the previous task's `register`ed variable, it's possible that task could change while this one doesn't, and Jenkins won't be immediately restarted.

We could either merge these two tasks into one with `with_items`, or use two registered variables to flag the restart with an `or`—I'd prefer the former if possible, to keep all the init changes in one task.